mailbox/store/src/test/java/org/apache/james/mailbox/store/AbstractMessageIdManagerQuotaTest.java

```diff
@@ -37,6 +37,7 @@
 import org.apache.james.mailbox.quota.CurrentQuotaManager;
```

Comment:
This class deserve to be in tools/quotas ?

Reply:
Not sure about this one?

Reply:
Not sure about this one?